### ISSUE TITLE
Need some kind of "quick start" page

### ISSUE THREAD

**Issue by willisblackburn** [java-json-tools/json-schema-validator#130]
I am using this validator on a project.  It's quite good, and I appreciate all the hard work put into it.

But I had a fair bit of trouble getting started with it simply because I am unable to find a page telling me how to actually instantiate a validator and perform basic validation.  There are samples, but the samples link actually links to the JavaDoc, not the code.  It's possible to find the samples and figure it out, but really it would be great to have an example on the main page, or some other easy-to-find page, showing basic usage.

**Comment by sisso** [comment]
I want to agree with @willisblackburn. I lost almost 30 minutes to find the javadocs, and then the class named Example1 and get the source. But only because I found this issue first.

A simple example in front page will really helps.

```java
import com.fasterxml.jackson.databind.JsonNode;
import com.fasterxml.jackson.databind.ObjectMapper;
import com.github.fge.jsonschema.core.report.ProcessingReport;
import com.github.fge.jsonschema.main.JsonSchema;
import com.github.fge.jsonschema.main.JsonSchemaFactory;

public final class Test
{
    public static void main(final String... args) throws Exception {
        ObjectMapper mapper = new ObjectMapper();
        JsonNode fstabSchema = mapper.readTree("{ \"type\": \"array\", \"items\": [ { \"type\": \"string\" } ], \"additionalProperties\": false }");
        JsonNode good = mapper.readTree("[\"one\",\"two\"]");
        JsonSchema schema = JsonSchemaFactory.byDefault().getJsonSchema(fstabSchema);
        ProcessingReport report = schema.validate(good);
        System.out.println(report);
    }
}
```

**Comment by fge** [comment]
@sisso eh? The README contains not only the links to the javadoc, but also to the code examples _and_ the maven artifacts; I really don't understand your point at all.

As to the library requiring a lot of dependencies, yes, it's a fact; I plan to change that for the next major version. What I don't know yet is whether this version will target Java 7 or Java 8. Maybe I'll release a last 2.x version with formats separated, since this is the part which requires the most dependencies.

**Comment by sisso** [comment]
Not everybody could agree that a javadoc is a good getting started (the sample, point to javadoc too). With a single example take seconds to undestand and started to use.

It is almost a pattern to have a sample in the README. Take some examples:

https://github.com/xetorthio/jedis
https://github.com/json4s/json4s

But is suggestion for new users, I already have my sample code.

**Comment by fge** [comment]
@sisso I _did_ mention that there was also a link to code examples, right?

Also, in practice you'll never initialize a schema from a string anyway.

**Comment by sisso** [comment]
I give up :P

**Comment by jmnarloch** [comment]
Yeah, the documentation is that what everyone one new to this project would appreciate, if you don't know even what is the API how you suppose to know from where to start.

If I find time, I could prepare really simple "5 minute..." getting started.

Also the link on the README for the "sample code" end with 404 response...

**Comment by ksl67** [comment]
I found this, which was very useful - http://wilddiary.com/validate-json-against-schema-in-java/

**Comment by ebuildy** [comment]
the demo project https://github.com/java-json-tools/json-schema-validator-demo (listed on README.md) gives a 404

**Comment by huggsboson** [comment]
forked the repo.  Making sure it's up to date.
